charts/unleash-edge/Chart.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: v2
+name: unleash-edge
+description: A Helm chart for deploying Unleash Edge to kubernetes
+
+type: application
+version: 0.1.0
+
+appVersion: "v0.1.0"
+maintainers:
+  - name: chriswk
+  - name: sighphyre
+  - name: nunogois
+  - name: gastonfournier
```

Do we care about newlines at the end of file? I have it set as my default (via .editorconfig), but I notice we don't do it a lot of places.

```suggestion
  - name: gastonfournier
  
```